docs/developer/app-store/apps/stripe/migration-from-plugin.mdx

```diff
@@ -0,0 +1,58 @@
+| Control of automatic/manual capture                     | Configured in plugin settings.<br/>If charge is automatic, it will be performed when the order is confirmed | App will use channel's setting Transaction Flow. Funds will be charged immediately in case of CHARGE                         |
+| Currency setting                                        | Configured in settings                                                                                      | Use Checkout/Order currency (inherited from Channel)                                                                         |
+| Passing customer email to Stripe                        | Configurable in plugin settings                                                                             | Not supported                                                                                                                |
+| Stripe authentication                                   | Plugin uses legacy Secret Key                                                                               | App uses Restricted Key                                                                                                      |
+| Stripe API version                                      | `2020-08-27`                                                                                                | `2025-04-30.basil`                                                                                                           |
+| Releasing funds if Order can't be created from checkout | Plugin will release the funds immediately.                                                                  | App will not refund funds, but [Saleor will eventually](../../../checkout/lifecycle#releasing-funds-for-abandoned-checkouts) |
+| Saving payment method in Stripe                         | Supported                                                                                                   | Not supported                                                                                                                |
+
+
+## How to migrate
+
+### Fetching configuration
+
+The plugin uses the `availePaymentGateways` query to fetch the configuration of the payment gateway. The App uses the `paymentGatewayInitialize` mutation to fetch the configuration.
+
+### Payment mutations
+
+The Plugin uses `checkoutPaymentCreate` mutation to create a payment. The App uses `transactionInitializeSession` and `transactionProcessSession` mutations to create a payment.
+
+See the [integration docs](./storefront-integration) for more details.
+
+### Running `checkoutComplete`
+
+The `checkoutComplete` mutation should be now run at the end of the process, when transactions are reported.
+
+The `paymentData` and `redirectUrl` input should not be used anymore, to transfer data to the App, use `data` fields on the transaction webhooks.
```

Comment:
`redirectUrl` is not payment redirect, it is order-detail redirect 👇 
> "URL of a view where users should be redirected to "
"see the order details. URL in RFC 1808 format."